src/components/Chat.jsx

```diff
@@ -16,7 +16,7 @@ const ChatApp = (props) => {
         <div className={styles.App}>
           <SendBirdApp
             appId={process.env.REACT_APP_SENDBIRD_ID}
-            userId={props.user.email}
+            userId={props.user.userId}
```

Comment:
@euniceyoh Is this change ok? Since `props.user.email` might be incorrect unless `props.user.emailVerified` is true.

Reply:
I changed sendbird's user's id to a user's name, even though they're not guaranteed to be unique, bc i needed to get the PT's email through marketplace, and it wasn't part of the schema.